### DISCUSSION TITLE
Appreciation

### DISCUSSION THREAD

**Discussion by pjaborges** [ERGO-Code/HiGHS#1063] [General]
I just recently start using HiGHS and I am very pleased with many aspects of it. 
Still a lot to explore such as the MIP solver and other possibilities to interact with it.
 
I would like to leave a word of massive appreciation for those that have put this solver together!

Thank you!

**Comment by jajhall** [comment]
Thanks. I'll reply and flag this as "answered"

**Comment by Philip-S-Martin** [comment]
Going to second this. Plus the attention paid to highspy lately has been incredible. Previously I had to build everything from scratch. Now it's easier than installing GLPK!

`pip install highspy`
`python -m pyomo.build-extensions`
